src/printer/index.ts

```diff
@@ -59,17 +89,18 @@ export default class HTMLPrinter implements Generatable<DMMFDocument> {
   <head>
     ${this.getHead('styles/main.css')}
   </head>
-  <body class="bg-gray-200">
+  <body class="bg-gray-200 dark:bg-black">
     <div class="flex min-h-screen">
       <div
-        class="sticky top-0 w-1/5 flex-shrink-0 h-screen p-4 px-6 mr-4 overflow-auto bg-white mac-h-screen"
+        class="sticky top-0 w-1/5 flex-shrink-0 h-screen p-4 px-6 overflow-auto bg-white dark:bg-black mac-h-screen"
```

Comment:
I would like to have same bg color for sidebar and content so match them to gray-800